src/main/java/com/autotune/analyzer/services/CreateExperiment.java

```diff
@@ -124,6 +127,8 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
             e.printStackTrace();
             LOGGER.error("Unknown exception caught: " + e.getMessage());
             sendErrorResponse(response, e, HttpServletResponse.SC_INTERNAL_SERVER_ERROR, "Internal Server Error: " + e.getMessage());
+        } finally {
+            createExp.stop(MetricsConfig.timercreateExp);
```

Comment:
Please check for null

if (null != createExp) createExp.stop(MetricsConfig.timercreateExp);